### PR TITLE
Merge Branch-UG3 Hong Liang with master

### LINKED ISSUE #156: [PE-D][Tester E] Command summary for `add` is not yet updated
I believe it should be `addPerson` instead of `add`.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: sugiyem/ped#1

### PR DESCRIPTION
**Pull request by tanhl2000**
closes #156, #162, #182

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -134,27 +128,15 @@ Format: `editPerson INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [g/GENDER] [d
...
+- The genders accepted by the contact list are: `M`/`m`/`Male`/`male`
+for male, `F`/`f`/`Female`/`female` for female.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be
+*  `editPerson 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be
    `91234567` and `johndoe@example.com` respectively.
-*  `edit 3 n/Charlotte g/F d/3/4/1998` Edits the 3rd person’s contact: edits name to be `Charlotte`,
+*  `editPerson 3 n/Charlotte g/F d/3/4/1998` Edits the 3rd person’s contact: edits name to be `Charlotte`,
 edits gender to be `Female` and edits date of birth to be `3.4.1998`.
```
Comment (Qiaoran-M) [inline docs/UserGuide.md]:
```suggestion
edits gender to be `Female` and edits date of birth to be `3/4/1998`.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -134,27 +128,15 @@ Format: `editPerson INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [g/GENDER] [d
 - Existing values will be updated to the input values.
 - At least one of the optional fields must be provided.
 - Date format accepted is: dd/mm/yyyy.
```
Comment (Qiaoran-M) [inline docs/UserGuide.md]:
```suggestion
- Date format accepted is: `dd/mm/yyyy`.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -85,18 +85,12 @@ Format: `addPerson n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS g/GENDER d/DOB`
 
 - The compulsory parameters are: name (`n`), phone number (`p`), email (`e`), address (`a`), gender (`g`), date of birth(`d`).
 - Date format accepted is: dd/mm/yyyy.
```
Comment (Qiaoran-M) [inline docs/UserGuide.md]:
```suggestion
- Date format accepted is: `dd/mm/yyyy`.
```

**Review by Qiaoran-M — CHANGES_REQUESTED** [review]
LGTM except for minor changes 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (tanhl2000) [reply docs/UserGuide.md]: thx for catching

File: docs/UserGuide.md (hunk 2 above)
Comment (tanhl2000) [reply docs/UserGuide.md]: will change

File: docs/UserGuide.md [hunk 4]
```diff
@@ -328,10 +310,10 @@ in the event list.
 
 ### Generating pie charts of statistics of the people tagged to an event in the event list : `makeStats`
 
-Format: `makeStats EVENT_INDEX t/STATISTIC_TYPE`
+Format: `makeStats INDEX t/STATISTIC_TYPE`
 
-* The `EVENT_INDEX` refers to the index number shown in the displayed event list.
-* The `EVENT_INDEX` must be **a positive integer** 1, 2, 3, …, and it must be within the range of the event list index. This command is invalid if `EVENT_INDEX` is a non-positive integer.
+* The `INDEX` refers to the index number shown in the displayed event list.
+* The `INDEX` must be a positive integer 1, 2, 3, …, and it must be within the range of the event list index.
```
Comment (Qiaoran-M) [inline docs/UserGuide.md]: This line overwrites the previous PR changes about invalid index.
```suggestion
* The `INDEX` must be **a positive integer** 1, 2, 3, …, and it must be within the range of the event list index. This command is invalid if `EVENT_INDEX` is a non-positive integer.
```

**Review by Qiaoran-M — CHANGES_REQUESTED** [review]
All good except for 1 minor change 👍

File: docs/UserGuide.md (hunk 4 above)
Comment (tanhl2000) [reply docs/UserGuide.md]: ah my mistake ok will change

**Review by Qiaoran-M — APPROVED** [review]
LGTM!

**Pull request merged**